autoscheduler/scraper/models/section.py

```diff
@@ -12,29 +14,38 @@ class Section(models.Model):
     course_num = models.IntegerField(db_index=True)
     section_num = models.IntegerField(db_index=True)
     term_code = models.IntegerField(db_index=True)
-
     min_credits = models.IntegerField() # Will never be null
     max_credits = models.IntegerField(null=True) # Will be null in most cases
-
     max_enrollment = models.IntegerField()
     current_enrollment = models.IntegerField()
     instructor = models.ForeignKey('Instructor', on_delete=models.CASCADE)
 
     class Meta:
         db_table = "sections"
 
+def generate_meeting_id(section_num: str, meetings_count: str):
+    """ Generates the meeting id in the form of {section_num}{meetings_count}"""
+
+    return "".join((section_num, meetings_count))

```

Comment:
This needs to be the section's `id` (not its crn) + `meeting_count`. Since most section numbers start at 500, this will very quickly run into intersections with other data.